samples/browser/src/main/java/org/mozilla/samples/browser/addons/AddonDetailsActivity.kt

```diff
@@ -105,3 +141,11 @@ class AddonDetailsActivity : AppCompatActivity() {
         return DateFormat.getDateInstance().format(formatter.parse(text)!!)
     }
 }
+
+private fun AddonUpdater.Request.getDialogMessage(context: Context): String {
+    val statusString = status.toLocalizedString(context)
+    val dateString = DateFormat.getDateTimeInstance(DateFormat.LONG, DateFormat.LONG).format(date)
+    val lastAttemptLabel = context.getString(R.string.mozac_feature_addons_updater_dialog_last_attempt)
+    val statusLabel = context.getString(R.string.mozac_feature_addons_updater_dialog_status)
+    return "$lastAttemptLabel: $dateString \n $statusLabel: $statusString ".trimMargin()
```

Comment:
Why are you hard-coding the colon? Please never hardcode text, put it into the strings.

Reply:
👍